R/data.R

```diff
@@ -61,3 +61,11 @@
 #' @keywords dataset
 #' @family dataset
 "vs"
+
+#' ADSL vaccine Dataset
+#'
+#' An ADaM dataset
+#'
+#' @keywords dataset
+#' @family dataset
+"adsl_vax"
```

Comment:
Well we have three adsl and now a fourth - lol @pharmaverse/admiralvaccines @neetusan  we need some guidance here/decision.  Causing issues.